balboa-http/src/main/scala/com/socrata/balboa/server/EntityJSON.scala

```diff
@@ -0,0 +1,5 @@
+package com.socrata.balboa.server
+
+case class EntityJSON(timestamp: Long, metrics: Map[String, MetricJSON])
```

Comment:
This case class represents precisely the entity JSON encoded in the request body of the POST request. This allows Jackson / Json4s to convert directly to and from JSON with out any extra marshalling methods, similar to how Rojoma Json classes were being used in the old balboa-http. Also note that it holds a timestamp, something not contained in the Metrics object.